docker-compose.yml

```
@@ -2,7 +2,23 @@ version: '3'
 
 services:
   db:
-    image: postgres:10.5
+    build:
+      context: ./db
+    volumes:
+      - postgres-volume:/var/lib/postgresql/data
+    env_file:
+      - .env.keycloak
+
```

```suggestion
    ports:
      - 5432
```
this ensure is does not clash with a local postgres instance